test/PooledStaking/updateParameter.js

```diff
@@ -0,0 +1,69 @@
+const { accounts, contract, web3 } = require('@openzeppelin/test-environment');
+const { BN, constants, expectEvent, expectRevert, ether } = require('@openzeppelin/test-helpers');
+const { assert } = require('chai');
+
+const { Role, ParamType } = require('../utils/constants');
+
+const MasterMock = contract.fromArtifact('MasterMock');
+const PooledStaking = contract.fromArtifact('PooledStaking');
+
+const [
+  nonMember,
+  member,
+  abMember,
```

Comment:
how about `advisoryBoardMember` ? i noticed a lot of tests in solidity world use these abbreviations, but with an IDE typing it shouldn't be an issue